clappr/clappr-ios
# Seekbar Plugin

### Goal

Creating a Seekbar plugin to append in MediaControl plugins.

### How to test

Run example and tests: 

Execute the video and make sure that you can interact with the seekbar. Try to execute multiples scenarios as seek a playing video, seek a paused video, enter and exit fullscreen, use VOD and Live videos and so on.
